### PR TITLE
feat: adds metaList

### PR DESCRIPTION
**Pull request by stutirao**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/atoms/metaList/Meta.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import Text from '@/components/atoms/text';
+import classNames from 'classnames';
+import { BaseProps, extractBaseProps } from '@/utils/types';
+import Icon from '@/components/atoms/icon';
```
Comment (aditya-kumawat) [inline core/components/atoms/metaList/Meta.tsx]: import {Icon, Text} from '@/index';

File: core/components/atoms/metaList/Meta.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+export interface MetaProps extends BaseProps {
+  /**
+   * Describes label of the `Meta data`
+   */
+  label: string;
+  /**
+   * Icon
```
Comment (aditya-kumawat) [inline core/components/atoms/metaList/Meta.tsx]: Update description

File: core/components/atoms/metaList/Meta.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    label,
+    icon,
+    className
+  } = props;
+
+  const baseProps = extractBaseProps(props);
+
+  const MetaClass = classNames({
+    ['Meta']: true,
+    ['bg-light']: true,
```
Comment (aditya-kumawat) [inline core/components/atoms/metaList/Meta.tsx]: Not required.

File: core/components/atoms/metaList/MetaList.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+export const MetaList = (props: MetaListProps) => {
+
+  const { list, className, seperator } = props;
+  const baseProps = extractBaseProps(props);
+
+  const MetaListClass = classNames({
+    ['Meta']: true,
+    ['bg-light']: true,
```
Comment (aditya-kumawat) [inline core/components/atoms/metaList/MetaList.tsx]: Not required.

File: core/components/atoms/metaList/MetaList.tsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+      {list.map((item, ind) => {
+        const { label = '', icon } = item;
+        const rightSeperator = ind === ((list.length) - 1) ? false : true;
+        return (
+          <span key={ind} className="MetaList-item Meta">
+            <Meta
+              label={label}
+              icon={icon}
+            />
+            {rightSeperator && (<span className={'Meta-seperator'} />)}
```
Comment (aditya-kumawat) [inline core/components/atoms/metaList/MetaList.tsx]: Meta-separator => MetaList-separator

File: core/components/atoms/metaList/MetaList.tsx [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+  const MetaListClass = classNames({
+    ['Meta']: true,
+    ['bg-light']: true,
+  }, className);
+
+  return (
+    <div {...baseProps} className={MetaListClass}>
+
+      {seperator && (<span className={'Meta-seperator Meta-leftSeperator'} />)}
```
Comment (aditya-kumawat) [inline core/components/atoms/metaList/MetaList.tsx]: Meta-leftSeparator => MetaList-separator--left

File: core/components/atoms/metaList/MetaList.tsx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+  seperator?: boolean;
+}
+
+export const MetaList = (props: MetaListProps) => {
+
+  const { list, className, seperator } = props;
+  const baseProps = extractBaseProps(props);
+
+  const MetaListClass = classNames({
+    ['Meta']: true,
```
Comment (aditya-kumawat) [inline core/components/atoms/metaList/MetaList.tsx]: Meta => MetaList

File: core/components/atoms/metaList/MetaList.tsx [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+  return (
+    <div {...baseProps} className={MetaListClass}>
+
+      {seperator && (<span className={'Meta-seperator Meta-leftSeperator'} />)}
+
+      {list.map((item, ind) => {
+        const { label = '', icon } = item;
+        const rightSeperator = ind === ((list.length) - 1) ? false : true;
+        return (
+          <span key={ind} className="MetaList-item Meta">
```
Comment (aditya-kumawat) [inline core/components/atoms/metaList/MetaList.tsx]: Remove Meta class from here and add required style in MetaList-item class only.

**Comment by aditya-kumawat** [comment]
@stutirao Please also include MetaList in the build.

**Comment by aditya-kumawat** [comment]
Update PR title.

File: core/components/atoms/metaList/Meta.tsx [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+  }, className);
+
+  return (
+    <div
```
Comment (aditya-kumawat) [inline core/components/atoms/metaList/Meta.tsx]: replace div with span

**Comment by aditya-kumawat** [comment]
Rename metaData.css => metaList.css

**Review by aditya-kumawat — DISMISSED** [review]

**Review by aditya-kumawat — APPROVED** [review]

**Review by satyamyadav — APPROVED** [review]

**Pull request merged**